toolchain/sem_ir/type_iterator.cpp

```diff
@@ -236,4 +235,7 @@ auto TypeIterator::PushInstId(SemIR::InstId inst_id) -> void {
 // Push the next step into the work queue.
 auto TypeIterator::Push(WorkItem item) -> void { work_list_.push_back(item); }
 
+static_assert(Carbon::Internal::Kind::IsStdVariant<std::variant<int, bool>>);
+static_assert(!Carbon::Internal::Kind::IsStdVariant<int>);
```

Comment:
Is this debug code? I'm not clear why it'd be here.

Reply:
Uh.. yeah thank you. I don't even remember this code, but anyway - removing it from the PR that was adding it.